README.md

```diff
@@ -3,37 +3,17 @@
 [![codecov](https://codecov.io/gh/jellyfish-dev/jellyfish/branch/main/graph/badge.svg?token=ANWFKV2EDP)](https://codecov.io/gh/jellyfish-dev/jellyfish)
 [![CircleCI](https://circleci.com/gh/jellyfish-dev/jellyfish.svg?style=svg)](https://circleci.com/gh/jellyfish-dev/jellyfish)
 
-## Usage
+Jellyfish is an open-source, general-purpose media server that ships with support for multiple media protocols.
+It can be thought of as a multimedia bridge meant for creating different types of multimedia systems that lets 
+you easily create a real-time video conferencing system, a broadcasting solution or both at the same time.
 
-### Running with Docker
+It leverages the [Membrane RTC Engine](https://github.com/jellyfish-dev/membrane_rtc_engine), a real-time communication engine/SFU library built with [Membrane](https://membrane.stream/).
 
-To download Jellyfish Docker image, see [Jellyfish images](https://github.com/jellyfish-dev/jellyfish/pkgs/container/jellyfish).
+## Documentation
 
-You can also build the image locally
+Everything you need to get started with Jellyfish is available in the [Jellyfish docs](https://jellyfish-dev.github.io/jellyfish-docs/).
```

Comment:
This makes the README quite empty. Some kind of "quickstart" would be useful.

1. I'd left the Installation guide with one command and a link to the proper docs section.
2. Another section ("Usage"?) could link to SDKs
3. In future, some examples could be linked here as well

These should be enough to get started ;)